src/libraries/Common/tests/TestUtilities/System/Runtime/InteropServices/SafeBufferUtil.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.InteropServices
+{
+    public static class SafeBufferUtil
+    {
+        /// <summary>
+        /// Creates an unmanaged buffer of the specified length.
+        /// </summary>
+        public static SafeBuffer CreateSafeBuffer(nuint byteLength)
+        {
+            return new AllocHGlobalSafeHandle(byteLength);
+        }
+
+        private sealed class AllocHGlobalSafeHandle : SafeBuffer
+        {
+            public AllocHGlobalSafeHandle(nuint cb) : base(ownsHandle: true)
+            {
+#if !NETCOREAPP
+                RuntimeHelpers.PrepareConstrainedRegions();
+#endif
```

Comment:
This is just a test file... is this really necessary?

Reply:
No. But everything is non-shipping code until it gets copied \& pasted into a shipping product. :)